app/frontend/src/components/NotificationBar/_NotificationBar.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+.notification-bar {
+  height: 55px;
```

Comment:
It looks like this was not used, which makes the component not match figma. Any specific reason why this was not used?